client_src/js/services/octofarm-client.service.js

```diff
@@ -99,27 +98,27 @@ export default class OctoFarmClient {
   }
 
   static async get(path) {
-    const url = new URL(this.base + path, window.location.origin).pathname;
+    const url = new URL(path, window.location.origin).href;
```

Comment:
These are better as pathname to support reverse proxy instances...

Reply:
Okido will revert

Reply:
I've just double checked the documentation and pathname won't work if we ever want to send `?=` queries so can be left as href as I think I was wrong with my reasoning before. Javascript should just detect what's in the url bar regardless.

Reply:
Yeah note I removed the params static. Got a bit of a different implementation in mind where each function like getPrinter(id)
constructs its own url from start to finish `${base}/${path}?param1=asd`
Just thought to let you know.